docs/source/index.rst

```diff
@@ -56,7 +56,6 @@ Daml Documentation
    :caption: Operating Daml
 
    operating-daml
-   ops/requirements
```

Comment:
Why was this line removed?

Reply:
I moved it to `operating-daml.rst`, bad idea ?

Reply:
I don't think it's a bad idea, thanks for clarifying.